src/components/chart/ChartContainer.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import * as _ from 'underscore';
+
+export interface ChartContainerProps {
+    id: string;
+    renderChart: (width: number, height: number) => React.ReactNode;
+}
+
+export interface ChartBaseProps {
+    width: number;
+    height: number;
+}
+
+export const ChartContainer: React.FunctionComponent<ChartContainerProps> = (props) => {
+    const container = React.useRef<HTMLDivElement>(null);
+    useDebouncedWindowResize();
+
+    const toggleSvgs = (show: boolean) => {
+        if (container.current) {
+            const svgs = container.current.querySelectorAll('svg');
+            _.each(svgs, (svg: SVGElement) => svg.style.display = show ? 'block' : 'none');
+        }
+    };
+
+    toggleSvgs(false);

```

Comment:
why we need to toggle hidden and show after ? I think if we need to calculate something on render, its better to use a component instead of a function

Reply:
I need to hide the svg to make sure It doesn't create scrollbar on the content (making the calculation wrong)